code/modules/projectiles/guns/smgs.dm

```diff
@@ -76,8 +76,8 @@
 	accuracy_mult_unwielded = 0.85
 	recoil_unwielded = 0
 	scatter = 0
-	fire_delay = 0.2 SECONDS
-	scatter_unwielded = 10 //Made to be used one handed.
+	fire_delay = 0.15 SECONDS
+	scatter_unwielded = 0 //Made to be better used one handed.
```

Comment:
i think 5 is a fair number, you can still use suppressor/recoil compensator with it so its bearable
```suggestion
	scatter_unwielded = 5 //Made to be better used one handed.
```

Reply:
A problem encountered during testing using simple ingame varediting was that any unwielded scatter above 0 would cause the scatter to act like it was 100% scatter chance for some heretofore unknown reason, perhaps there was a var that made that so, but if there is, we don't know which one it is. Or perhaps it was a problem connected with ingame varediting?